### PR TITLE
added a UUID generator

### PR DESCRIPTION
**Pull request by FlorianZipser**
I added the UuidGenerator class to randomly generate UUIDs and a test for it. Further I registered the generator at the bean-matcher class but unfortunatlely reformatted the class :(.  It would be great if you could consider to merge the pull request.

### CODE REVIEW

File: src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+	private UuidGenerator unitUnderTest;
+
+	@BeforeMethod
+	public void setUp() {
+		unitUnderTest = new UuidGenerator();
+	}
+
+	@Test
+	public void shouldProvideString() {
```
Comment (orien) [inline src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java]: Rename to `shouldProvideUuid`.

File: src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		// when
+		UUID result = unitUnderTest.generate();
+
+		// then
+		assertThat(result, is(notNullValue()));
+	}
+
+	@Test
+	public void shouldProvideDifferingStringOnSubsequentCall() {
```
Comment (orien) [inline src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java]: Rename to `shouldProvideDifferingUuidOnSubsequentCall`.

File: src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java (hunk 1 above)
Comment (orien) [inline src/test/java/com/google/code/beanmatchers/UuidGeneratorTest.java]: Rename to `shouldProvideUuid`.

**Comment by FlorianZipser** [comment]
Sorry for the reformatting. I reverted them and disabled my Eclipse settings. Further I renamed the test. Please let me know, if there are further issues.

**Comment by orien** [comment]
Thanks @FlorianZipser. Re the indentation: could you please replace the tabs with two spaces?

**Comment by FlorianZipser** [comment]
I replaced the tabs with blanks and formatted both classes UuidGenerator and UuidGeneratorTest with a code style for Eclipse I found at https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml. Is this the proper style?

**Pull request merged**